packages/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.4.1
+
+* **Breaking change**. The `pickImage` method will now return null when the user cancels picking the image or denies the needed permissions.
```

Comment:
Correct me if I'm wrong, but the previous way was actually ignoring the `RESULT_CANCELED` alltogether and discarding the `pendingResult`:

* [User canceled picking images from gallery](https://github.com/flutter/plugins/blob/master/packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerPlugin.java#L129)
* [User canceled taking images with camera](https://github.com/flutter/plugins/blob/master/packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerPlugin.java#L149)

Reply:
lol. You're right, it would just hang I guess. Thanks for fixing that.

I think this is just a bugfix, we can leave the version number as 0.4.1 and you can remove the **breaking change** from the changelog.